start/marleybot.js

```diff
@@ -17,51 +17,65 @@ limitations under the License.
 This is a sample Slack bot built with Botkit.
 */
 
-const { Botkit } = require('botkit')
-const { SlackAdapter, SlackEventMiddleware } = require(
-  'botbuilder-adapter-slack')
-const { SecretManagerServiceClient } = require('@google-cloud/secret-manager')
+const { Botkit } = require("botkit");
+const {
+  SlackAdapter,
+  SlackEventMiddleware,
+} = require("botbuilder-adapter-slack");
+const { SecretManagerServiceClient } = require("@google-cloud/secret-manager");
+
+const ConduitClient = require("./conduitClient.js");
+const BASE_PHABRICATOR_URL = "https://phabricator.khanacademy.org";
 
 /**
  * Returns the secret string from Google Cloud Secret Manager
  * @param {string} name The name of the secret.
  * @return {string} The string value of the secret.
  */
-async function accessSecretVersion (name) {
-  const client = new SecretManagerServiceClient()
-  const projectId = process.env.PROJECT_ID
+async function accessSecretVersion(name) {
+  const client = new SecretManagerServiceClient();
+  const projectId = process.env.PROJECT_ID;
   const [version] = await client.accessSecretVersion({
-    name: `projects/${projectId}/secrets/${name}/versions/1`
-  })
+    name: `projects/${projectId}/secrets/${name}/versions/1`,
+  });
 
   // Extract the payload as a string.
-  const payload = version.payload.data.toString('utf8')
+  const payload = version.payload.data.toString("utf8");
 
-  return payload
+  return payload;
 }
 
 /**
  * Asynchronous function to initialize marleybot.
  */
-async function marleybotInit () {
+async function marleybotInit() {
   const adapter = new SlackAdapter({
-    clientSigningSecret: await accessSecretVersion('client-signing-secret'),
-    botToken: await accessSecretVersion('bot-token')
-  })
+    clientSigningSecret: await accessSecretVersion("client-signing-secret"),
+    botToken: await accessSecretVersion("bot-token"),
+  });
 
-  adapter.use(new SlackEventMiddleware())
+  adapter.use(new SlackEventMiddleware());
 
   const controller = new Botkit({
-    webhook_uri: '/api/messages',
-    adapter: adapter
-  })
+    webhook_uri: "/api/messages",
+    adapter: adapter,
+  });
+
+  const conduitAPIToken = await accessSecretVersion("conduit-api-token");
```

Comment:
This is the real part. I'd like to improve testing for our bot so I don't have to deploy and then test 😆